src/pages/safety-&-side-effects.astro

```diff
@@ -1,18 +1,270 @@
+              </div>
+
+              <div class="side-effects-page--commun-list__item">
+                <img class="side-effects-page--commun-list__item__img" src="/assets/images/safety-and-side-effects/icons/muscle-pain-icon.png" alt=" " srcset="">
+                <span class="side-effects-page--commun-list__point desktop--only"></span>
+                Muscle and bone pain
+              </div>
+
+            </div>
+
+            <div class="side-effects-page--commun-list__column">
+              
+              <div class="side-effects-page--commun-list__item">
+                <img class="side-effects-page--commun-list__item__img" src="/assets/images/safety-and-side-effects/icons/fever-icon.png" alt=" " srcset="">
+                <span class="side-effects-page--commun-list__point desktop--only"></span>
+                Fever
+              </div>
+
+              <div class="side-effects-page--commun-list__item">
+                <img class="side-effects-page--commun-list__item__img" src="/assets/images/safety-and-side-effects/icons/mouth-icon.png" alt=" " srcset="">
+                <span class="side-effects-page--commun-list__point desktop--only"></span>
+                Inflammation of the mouth
+              </div>
+
+              <div class="side-effects-page--commun-list__item">
+                <img class="side-effects-page--commun-list__item__img" src="/assets/images/safety-and-side-effects/icons/headache-icon.png" alt=" " srcset="">
+                <span class="side-effects-page--commun-list__point desktop--only"></span>
+                Headache
+              </div>
+
+              <div class="side-effects-page--commun-list__item">
+                <img class="side-effects-page--commun-list__item__img" src="/assets/images/safety-and-side-effects/icons/finger-icon.png" alt=" " srcset="">
+                <span class="side-effects-page--commun-list__point desktop--only"></span>
+                Redness around the fingernails
+              </div>
+
+              <div class="side-effects-page--commun-list__item">
+                <img class="side-effects-page--commun-list__item__img" src="/assets/images/safety-and-side-effects/icons/itching-icon.png" alt="" srcset="">
+                <span class="side-effects-page--commun-list__point desktop--only"></span>
+                Itching
+              </div>
+
+            </div>
+
+          </div>
+
+          <h3 class="side-effects-page--section__heading side-effects-page--section__heading__3">These are not all the possible side effects of Koselugo</h3>
+
+          <p class="side-effects-page--section__paragraph">
+            For more information, ask your doctor or pharmacist. Call your doctor for medical advice about side effects. You may report side effects to the FDA at <a class="side-effects__link" href="tel:1-800-FDA-1088">1-800-FDA-1088</a> .
+          </p>
+
+        </div>
+        
+      </Section>
+
+      <section class="side-effects-page--secondary-section">
```

Comment:
In tablet versions, this sections appears outside the section above.

## Site
![image](https://user-images.githubusercontent.com/31933833/228647154-274c4c27-2373-480c-84e0-bf943b2a9fa6.png)

## Figma
![image](https://user-images.githubusercontent.com/31933833/228647311-662fe346-1b7d-4680-a162-45bd4a0dae2d.png)